tests/python/unittest/test_tensorrt.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+from unittest import mock
+import tempfile
+
+import tvm
+
+def test_empty_library_export():
+    temp_dir = None
+    temp_file_path = None
+    try:
```

Comment:
Instead of using `try...catch`, can you use the following so that you don't need to take care of the cleanup process?
```python
with tempfile.TemporaryDirectory():
    ...
```